metrics/exporter.go

```diff
@@ -121,6 +121,12 @@ func isNewExporterRequired(newConfig *metricsConfig) bool {
 		return true
 	}
 
+	// If the OpenCensus address has changed, restart the collector.
+	// TODO(evankanderson): Should we just always restart the opencensus agent?
+	if newConfig.backendDestination == OpenCensus && newConfig.collectorAddress != cc.collectorAddress {
```

Comment:
&& newConfig.requiresSecure != cc.requiresSecure?

Reply:
Broke this up, since it should be an `||` not an `&&`.